ui/components/app/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -46,7 +50,7 @@ const REASON_TO_TITLE_TKEY = Object.freeze({
   [BlockaidReason.rawSignatureFarming]: 'blockaidTitleSuspicious',
 });
 
-function BlockaidBannerAlert({ securityAlertResponse, ...props }) {
+function BlockaidBannerAlert({ securityAlertResponse, txData, ...props }) {
```

Comment:
since we're passing txData now, can we remove `securityAlertResponse` from the params?